freeipa/src/main/resources/freeipa-salt/salt/freeipa/templates/initial-ldap-conf.ldif

```diff
@@ -1,4 +1,22 @@
 dn: cn=config,cn=ldbm database,cn=plugins,cn=config
 changetype: modify
 replace: nsslapd-db-locks
-nsslapd-db-locks: 200000
\ No newline at end of file
+nsslapd-db-locks: 200000
+
+
+dn: cn=config
+changetype: modify
+add: nsslapd-sizelimit
```

Comment:
maybe use `replace` here instead of `add` to match the other configs you are setting? It's not a big deal since this is the first time we are setting this config

Reply:
let me try it out first